### PR TITLE
Finished audition page mobile design

### PR DESCRIPTION
**Pull request by TheHiddenLayer**

### CODE REVIEW

**Review by jayrc7 — CHANGES_REQUESTED** [review]
This could be an issue that only shows up on my end but for some reason when trying to view the audition page using an ipad screen, the page won't show up at all.

**Review by jayrc7 — CHANGES_REQUESTED** [review]
Confirmed today in our meeting, the website shows a blank screen when viewing the audition page with an ipad. It's pretty much a css issue.

**Review by TheHiddenLayer — COMMENTED** [review]
There's like 76 files changed. Disregard everything except for Audition.css and Audition.js. All the changes are because I recently got an auto-formatter (and it just scanned the entire repo and perfected the style). I forgot to turn off the formatter when working on our TSE repos, but on the bright side we have perfect style now!

File: src/css/Audition.css [hunk 1]
```diff
@@ -1,76 +1,85 @@
...
+  .dots {
+    display: none;
+  }
+  .audition-info {
+    margin-bottom: -20px;
+  }
+  .image2 {
+    margin-left: -8%;
+    width: 110%;
+  }
```
Comment (TheHiddenLayer) [inline src/css/Audition.css:69]: In conclusion, just direct your attention here. This is most of the important stuff I ended up changing.

**Review by jayrc7 — APPROVED** [review]
Looks great, Thanks Upo!

**Pull request merged**